src/backend/src/plugins/action/action_service_impl.h

```diff
@@ -162,11 +162,18 @@ class ActionServiceImpl final : public rpc::action::ActionService::Service {
     grpc::Status
     SetTakeoffAltitude(grpc::ServerContext * /* context */,
                        const rpc::action::SetTakeoffAltitudeRequest *request,
-                       rpc::action::SetTakeoffAltitudeResponse * /* response */) override
+                       rpc::action::SetTakeoffAltitudeResponse *response) override
     {
         if (request != nullptr) {
             const auto requested_altitude = request->altitude();
-            _action.set_takeoff_altitude(requested_altitude);
+            mavsdk::Action::Result action_result = _action.set_takeoff_altitude(requested_altitude);
```

Comment:
Yep! That's not related to `PositionNED`, but let's keep it in this PR for this time :+1:.